src/modm/driver/display/ili9341.hpp

```diff
@@ -280,7 +280,7 @@ class Ili9341 : public Interface, public modm::GraphicDisplay
 
 private:
 	void
-	setColoredPixel(int16_t x, int16_t y, glcd::Color const &color);
+	setColoredPixel(int16_t x, int16_t y, color::Rgb565 const &color);
```

Comment:
For small types (size <= 2..3 ints) pass by value is generally recommended: [F.16: For “in” parameters, pass cheaply-copied types by value and others by reference to const](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-in):

> When copying is cheap, nothing beats the simplicity and safety of copying, and for small objects (up to two or three words) it is also faster than passing by reference because it does not require an extra indirection to access from the function.